tests/app/controllers/test_party_controller.py

```diff
@@ -275,3 +278,25 @@ def test_display_button(self):
         ]
         for combination in combinations:
             self.assertEqual(display_button(combination.status, combination.ci_type), combination.expected)
+
+    def test_filter_ended_collection_exercises(self):
+        """Tests the functionality of the 'filter_ended_collection_exercises' function"""
+        with open('tests/test_data/party/collection_exercises.json') as business_json_data:
+            data = json.load(business_json_data)
+
+        # Data has 2 collection exercises in the far future, nothing should be removed
```

Comment:
perhaps a bit nitpicky but couldnt we mock datetime.now to return a static date? wont have to worry about the test failing in the fututre then